### PR TITLE
Adds 10mm caseless ammunition

### PR DESCRIPTION
**Pull request by ynot01**
# Document the changes in your pull request

Adds 10mm caseless ammo to the uplink and autolathe alongside 10mm and 10mm sopoforic

Drops no casing when fired, but does less damage (same as AP) and travels slower (bit faster than NT-USP)

# Spriting

image

# Wiki Documentation

27 damage, 0.55 bullet speed

# Changelog

:cl:  
rscadd: Added 10mm caseless ammunition to the uplink and autolathe, which drops no casings but does slightly less damage and travels slower
/:cl:

### CODE REVIEW

**Comment by yacabo** [comment]
This is a really neat idea

File: code/game/objects/items/storage/uplink_kits.dm [hunk 1]
```diff
@@ -567,6 +567,13 @@
 	for(var/i in 1 to 2)
 		new /obj/item/ammo_box/magazine/m10mm(src)
 
+/obj/item/storage/box/syndie_kit/pistolcaselessammo
+	real_name = "10mm caseless magazine box"
+
+/obj/item/storage/box/syndie_kit/pistolsleepyammo/PopulateContents()
```
Comment (Mqiib) [inline code/game/objects/items/storage/uplink_kits.dm]: wrong box, should be caseless not sleepy ammo

**Comment by MajManatee** [comment]
noob bait

**Review by cuackles — APPROVED** [review]

**Comment by cuackles** [comment]
boneless bullets

**Review by ToasterBiome — APPROVED** [review]

**Comment by DylonBR** [comment]
noob bait

**Comment by KingSuperBOB** [comment]
Hes firing the whole bullet

**Pull request merged**